src/T2DMap.cpp

```diff
@@ -2638,30 +2684,30 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             TRoom* room = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
             if (room) {
-                QAction* action = new QAction(tr("Undo", "Menu option to undo drawing the last segment of a line in the mapper"), this);
-                action->setToolTip(tr("Undo last point"));
+                auto customLineUndoLastPoint = new QAction(tr("undo", "2D Mapper context menu (drawing custom exit line) item"), this);
```

Comment:
Is the back-to-lowercase here and below intentional?

Reply:
It seems to have been - but it *does* look misguided - I'll put them back as "Initial capitals"... :confounded: